test/common/ssl/context_impl_test.cc

```diff
@@ -266,5 +266,17 @@ TEST_F(SslServerContextImplTicketTest, TicketKeyInlineFailTooSmall) {
   EXPECT_THROW(loadConfigV2(cfg), EnvoyException);
 }
 
+TEST_F(SslServerContextImplTicketTest, CRLSuccess) {
```

Comment:
Please also add a test case for invalid crl (empty file maybe?)

Reply:
Done!  I think an empty file is actually okay (list with 0 elements), so I added a file with invalid PEM data.